assets/php/class-wgpb-block-grid-base.php

```diff
@@ -230,6 +228,22 @@ protected function get_container_classes() {
 			$classes[] = "align{$this->attributes['align']}";
 		}
 
+		if ( empty( $this->attributes['contentVisibility']['title'] ) ) {
+			$classes[] = 'is-hidden-title';
+		}
+
+		if ( empty( $this->attributes['contentVisibility']['price'] ) ) {
+			$classes[] = 'is-hidden-price';
+		}
+
+		if ( empty( $this->attributes['contentVisibility']['rating'] ) ) {
+			$classes[] = 'is-hidden-rating';
+		}
+
+		if ( empty( $this->attributes['contentVisibility']['button'] ) ) {
+			$classes[] = 'is-hidden-button';
+		}
+
```

Comment:
I left these out intentionally– the CSS hiding was a hack around filtering the shortcode. If we're not showing this content, we should remove it from the rendered output.